### PR TITLE
🏗 Create design review issues automatically

### PR DESCRIPTION
**Pull request by alanorozco**
generated: https://github.com/alanorozco/amphtml/issues/36

### CODE REVIEW

File: .github/workflows/create-design-review-issues.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set Up Node
+        uses: actions/setup-node@v2.1.5
+
+      - name: Generate
+        id: generate
+        run: |
+          issue=$(wget -q -O - https://raw.githubusercontent.com/ampproject/amphtml/master/build-system/tasks/create-design-review-issues/index.js | node)
```
Comment (alanorozco) [inline .github/workflows/create-design-review-issues.yml:23]: Felt silly to checkout the repo / npm install for a standalone script, so we can just download it.

**Comment by kristoferbaxter** [comment]
You are an absolute saint for making this.

Will review first thing tomorrow, but thank you so much!

**Comment by dmanek** [comment]
This is awesome!

File: .github/workflows/create-design-review-issues.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+name: Create Design Review Issues
+
+on:
+  schedule:
+    # Every Wednesday at 00:00:00
+    - cron: '0 0 * * 3'
+
+jobs:
+  sweep-experiments:
```
Comment (samouri) [inline .github/workflows/create-design-review-issues.yml:9]: `sweep-experiments` doesn't seem quite right :)

File: .github/workflows/create-design-review-issues.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        id: generate
+        run: |
+          issue=$(wget -q -O - https://raw.githubusercontent.com/ampproject/amphtml/master/build-system/tasks/create-design-review-issues/index.js | node)
+          issue="${issue//'%'/'%25'}"
+          issue="${issue//$'\n'/'%0A'}"
+          issue="${issue//$'\r'/'%0D'}"
+          echo "::set-output name=issue::$(echo "$issue")"
+
+      - name: Create Issue
+        uses: nashmaniac/create-issue-action@v1.1
```
Comment (samouri) [inline .github/workflows/create-design-review-issues.yml:27]: just [checked out the repo](https://github.com/nashmaniac/create-issue-action) and it only has 1 star / very little activity. are you tied to this action?

it looks like there is a more popular one at [JasonEtco/create-an-issue](https://github.com/JasonEtco/create-an-issue)

File: build-system/tasks/create-design-review-issues/index.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+If you are interested in bringing your design to design review, read the [design review documentation](https://github.com/ampproject/amphtml/blob/master/contributing/design-reviews.md) and add a link to your design doc or issue by the Monday before your design review.
+
+When attending a design review please read through the designs _before_ the design review starts. This allows us to spend more time on discussion of the design.
+
+We rotate our design review between times that work better for different parts of the world as described in our [design review documentation](https://github.com/ampproject/amphtml/blob/master/contributing/design-reviews.md), but you are welcome to attend any design review. If you cannot make any of the design reviews but have a design to discuss please let mrjoro@ know on [Slack](https://github.com/ampproject/amphtml/blob/master/CONTRIBUTING.md#discussion-channels) and we will find a time that works for you.
+`.trim();
+
+function padleft(str, size) {
```
Comment (samouri) [inline build-system/tasks/create-design-review-issues/index.js:54]: nit: can we use [String.prototype.padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart)

File: build-system/tasks/create-design-review-issues/index.js [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+  str = str.toString();
+  while (str.length < size) {
+    str = '0' + str;
+  }
+  return str;
```
Comment (samouri) [inline build-system/tasks/create-design-review-issues/index.js:59]:
```suggestion
  return str.padStart(size, '0');
```

File: build-system/tasks/create-design-review-issues/index.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+  const resultDate = new Date(date.getTime());
+  resultDate.setDate(
+    resultDate.getDate() +
+      (weeks - 1) * 7 +
+      ((7 + dayOfWeek - date.getDay()) % 7)
+  );
+  return resultDate;
+}
+
+function getRotation(date, startYyyyMmDd) {
```
Comment (samouri) [inline build-system/tasks/create-design-review-issues/index.js:72]: nit/q: what is your method for choosing when to use the prefix `get`?

IMO either all of these side effect free value-returning functions should start with `get` or none of them should.

**Review by samouri — COMMENTED** [review]
Small nits. This looks great!

File: .github/workflows/create-design-review-issues.yml (hunk 1 above)
Comment (kristoferbaxter) [reply .github/workflows/create-design-review-issues.yml:20]: Clever.

File: .github/workflows/create-design-review-issues.yml (hunk 3 above)
Comment (kristoferbaxter) [reply .github/workflows/create-design-review-issues.yml:27]: Given how little code is present in these actions, might consider rolling your own for this.

But +1 to the suggestion here.

**Review by kristoferbaxter — APPROVED** [review]
Looks great overall, a few comments here and there.

File: .github/workflows/create-design-review-issues.yml (hunk 3 above)
Comment (alanorozco) [reply .github/workflows/create-design-review-issues.yml]: I just did what was easiest. But we can do a simple POST request instead, so rolling my own now. Bonus that the task time reduced from ~40s to ~4s.

**Pull request merged**